src/commands/governance/submit_proposal/authorize_to_subnet.rs

```diff
@@ -0,0 +1,85 @@
+use crate::lib::agent::create_waiter;
+use crate::lib::error::NnsCliResult;
+use crate::lib::nns_types::governance::governance_canister_id;
+
+use candid::{CandidType, Decode, Encode};
+use clap::Clap;
+use ic_agent::Agent;
+use ic_nns_common::pb::v1::NeuronId as NeuronIdProto;
+use ic_nns_common::types::NeuronId;
+use ic_nns_governance::pb::v1::manage_neuron_response::Command::{Error, MakeProposal};
+use ic_nns_governance::pb::v1::proposal::Action;
+use ic_nns_governance::pb::v1::NnsFunction::SetAuthorizedSubnetworks;
+use ic_nns_governance::pb::v1::{
+    manage_neuron::Command, ExecuteNnsFunction, ManageNeuron, ManageNeuronResponse, Proposal,
+};
+
+use ic_types::Principal;
+
+const MANAGE_NEURON_METHOD: &str = "manage_neuron";
+
+/// Submit a proposal to authorize a principal to one or more verified application subnetworks
+#[derive(Clap, Clone)]
+pub struct AuthToSubnetOpts {
```

Comment:
in general I\'m not sure whether it's the best option to keep these types separate. what happens when we evolve the types in the main repo

Reply:
is there a case to merge this tool to the main repo?